src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -18,6 +18,7 @@ namespace System.Runtime.Serialization
     using DataContractDictionary = System.Collections.Generic.Dictionary<System.Xml.XmlQualifiedName, DataContract>;
     using System.Linq;
     using System.Diagnostics.CodeAnalysis;
+    using System.Runtime.Serialization.Json;
 
```

Comment:
You now have unnecessary using's referencing S.R.S.Json as this was needed to pull in SerializerTrimmingWarning. I suspect this exists in many files.